web-wallet/README.md

```diff
@@ -37,23 +37,15 @@ VITE_FEATURE_ALLOCATE=true
 VITE_FEATURE_MIGRATE=true
 VITE_FEATURE_STAKE=true
 VITE_FEATURE_TRANSFER=true
-VITE_CURRENT_NODE=${VITE_LOCAL_NODE}
-VITE_CURRENT_PROVER_NODE=${VITE_LOCAL_NODE}
 VITE_GAS_LIMIT_DEFAULT=2900000000
 VITE_GAS_LIMIT_LOWER=10000000
 VITE_GAS_LIMIT_UPPER=1000000000
 VITE_GAS_PRICE_DEFAULT=1
 VITE_GAS_PRICE_LOWER=1
-VITE_LOCAL_NODE="https://localhost:5173/rusk/"
-VITE_MAINNET_NODE=""
 VITE_MIGRATE_CONTRACT=""
 VITE_MINIMUM_ALLOWED_STAKE=1000
 VITE_MODE_MAINTENANCE=false
-VITE_RKYV_TREE_LEAF_SIZE=632
-VITE_STAKE_CONTRACT="0200000000000000000000000000000000000000000000000000000000000000"
-VITE_TESTNET_NODE="https://nodes.dusk.network/"
-VITE_TESTNET_PROVER_NODE="https://provers.dusk.network/"
-VITE_TRANSFER_CONTRACT="0100000000000000000000000000000000000000000000000000000000000000"
+VITE_WALLET_NETWORK="https://nodes.dusk.network/"
```

Comment:
That's fine, but I think it's still worth noting in the README that this is the Mainnet link. People need to know that we also have the `https://testnet.nodes.dusk.network/`, and `https://devnet.nodes.dusk.network/`

Reply:
That bit of README is meant just as an example as it was before, but I agree with your point: open an issue to update the README to explain the different networks.